### PR TITLE
Add filter and limit by query parameter

### PR DESCRIPTION
**Pull request by edetec**
* Add Sqlalchemy filters by query string parameter;
* Add limit result by query string parameter;

### CODE REVIEW

File: flask_rest_orm/resources/utils.py [hunk 1]
```diff
@@ -22,10 +48,34 @@ def query_from_request(model, request):
     :return: the items for the current page
     """
     builder = model.query
-    for key in request.args:
-        if hasattr(model, key):
-            vals = request.args.getlist(key)  # one or many
-            builder = builder.filter(getattr(model, key).in_(vals))
+
+    def get_operator(attr, key, value):
```
Comment (igortg) [inline flask_rest_orm/resources/utils.py]: This function doesn't need to be an inner function
`attr` should be `model_attr`
`key` should be `operator_key`
`value` should be `filter_value`

File: flask_rest_orm/resources/utils.py [hunk 2]
```diff
@@ -22,10 +48,34 @@ def query_from_request(model, request):
...
+            if op == 'between':
+                return attr.between(value[0],value[1])
+            return getattr(attr, op)(value)
+        elif key in OPERATE:
+            op = OPERATE.get(key)
+            return attr.operate(getattr(operator, op), value)
+        else:
+            raise Exception('Unknown operator')
+
+    def filter(attr, value):
```
Comment (igortg) [inline flask_rest_orm/resources/utils.py]: `build_query` would be a better name for the function. `request_filter` a better name for `value`

File: flask_rest_orm/resources/utils.py [hunk 3]
```diff
@@ -22,10 +48,34 @@ def query_from_request(model, request):
...
+        elif key in OPERATORS:
+            op = OPERATORS.get(key)
+            if op == 'between':
+                return attr.between(value[0],value[1])
+            return getattr(attr, op)(value)
+        elif key in OPERATE:
+            op = OPERATE.get(key)
+            return attr.operate(getattr(operator, op), value)
+        else:
+            raise Exception('Unknown operator')
```
Comment (igortg) [inline flask_rest_orm/resources/utils.py]: Avoid use of generic `Exception`. Use `ValueError` and print the value of the input on the exc message

File: flask_rest_orm/resources/utils.py [hunk 4]
```diff
@@ -22,10 +48,34 @@ def query_from_request(model, request):
     :return: the items for the current page
     """
     builder = model.query
```
Comment (igortg) [inline flask_rest_orm/resources/utils.py]: A better name for this var would be `query`

File: flask_rest_orm/resources/utils.py [hunk 5]
```diff
@@ -1,5 +1,31 @@
...
+    'match': 'match',
+    'startswith': 'startswith',
+    'endswith': 'endswith',
+    'contains': 'contains',
+    'in': 'in_',
+    'notin': 'notin_',
+    'between': 'between',
+}
+
+OPERATE = {
```
Comment (igortg) [inline flask_rest_orm/resources/utils.py]: A better name would be `COMPARE_OPERATORS`. Would be also good to have some docs on why you did separate them in two lists

Comment (edetec) [reply flask_rest_orm/resources/utils.py]: OK

File: flask_rest_orm/resources/utils.py (hunk 4 above)
Comment (edetec) [reply flask_rest_orm/resources/utils.py]: OK

File: flask_rest_orm/resources/utils.py (hunk 1 above)
Comment (edetec) [reply flask_rest_orm/resources/utils.py]: OK

File: flask_rest_orm/resources/utils.py (hunk 3 above)
Comment (edetec) [reply flask_rest_orm/resources/utils.py]: OK

File: flask_rest_orm/resources/utils.py (hunk 2 above)
Comment (edetec) [reply flask_rest_orm/resources/utils.py]: OK

**Pull request merged**